Formula/libpostal.rb

```diff
@@ -0,0 +1,44 @@
+class Libpostal < Formula
+  desc "C library for parsing or normalizing addresses"
+  homepage "https://github.com/openvenues/libpostal"
+  url "https://github.com/openvenues/libpostal/archive/v0.3.1.tar.gz"
+  head "https://github.com/openvenues/libpostal.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "snappy" => :run
+  depends_on "pkg-config" => :run
```

Comment:
Is this actually used at run or is this a dance to be able to use it in the test?